client/my-sites/plan-features-2023-grid/components/plan-comparison-grid.tsx

```diff
@@ -744,6 +744,10 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 	const isLargeBreakpoint = usePricingBreakpoint( 1612 ); // 1340px + 272px (sidebar)
 	const isMediumBreakpoint = usePricingBreakpoint( 1340 ); // keeping original breakpoint to match Plan Grid
 
+	const isSignupLargeBreakpoint = usePricingBreakpoint( 1281 );
```

Comment:
We now have two sets of breakpoint state values here – one is used in the signup flow, and one is used in the administration page. However, they are both computed no matter which context we are in. Could we only compute the necessary breakpoint values for the specific context?

Reply:
That makes sense. I'll revisit the code

Reply:
Adjusted the hooks accordingly